index.html

```diff
@@ -163,9 +162,11 @@ <h4 class="scoial-text">LETS'S CONNECT</h4>
             <p class="lang">Langguages</p>
             <span class="material-icons icon-colors">expand_more</span>
           </div>
-          <img src="images/js-frame.png" alt="Missed Image" class="skill-img">
-          <img src="images/html-frame.png" alt="Missed Image" class="skill-img">
-          <img src="images/css-frame.png" alt="Missed Image" class="skill-img">
+          <div>
+            <img src="img/frames/js-frame.png" alt="Missed Image" class="skill-img">
+            <img src="img/frames/html-frame.png" alt="Missed Image" class="skill-img">
+            <img src="img/frames/css-frame.png" alt="Missed Image" class="skill-img">
+          </div>
           <div class="lang-skills">
           <p class="other-skill">Frameworks</p>
           <span class="material-icons icon-colors">chevron_right</span>
```

Comment:
_I will leave all the messages here regarding this review here_ 🌟🌟

- [ ] Hi, good job working on the project so far! I have noticed that the elements in the header are not links. Since the header contains the navigation system of the page then, lets's make sure that the user is able to click them to be able to go to other sections of the website 👍🏻. I kindly suggest that you add links to the `<li>` tags and to make the logo a link too.

- [ ] I have also noticed that the background image has failed to load for the Headline section. I suggest that you check the background image url so that it can load properly in your design. This concern also applies to the "Contact me section".

- [ ] Lastly, as per project requirement, 👇🏻 
<img width="359" alt="req" src="https://user-images.githubusercontent.com/4247740/144692337-bec58172-b183-4f6d-b922-5def36326e65.PNG">

- [ ] 👉🏻 When I try to move into the mobile view, everything gets distorted! I kindly request that you fix all elements as they should appear in mobile (as you did in the earlier projects before this one) and then apply media queries as required to load the desktop view - as the previous reviewer had requested!


⭐ _Good luck working on these changes!_ ⭐

Reply:
is the range between 992 to 1100 enough for the task ?

Reply:
actually, the project requirements didn't mention any mobile version, and even the previous reviewer didn't ask about it he asked for a desktop version especially with 1024px
![Att](https://user-images.githubusercontent.com/35707975/144714338-e2787efd-19e8-4374-9e94-66f241246b70.png)
they asked for between 992 to 1100 only. they